docs/topics/signals.rst

```diff
@@ -118,7 +118,11 @@ engine_started
 
 .. note:: This signal may be fired *after* the :signal:`spider_opened` signal,
     depending on how the spider was started. So **don't** rely on this signal
-    getting fired before :signal:`spider_opened`.
+    getting fired before :signal:`spider_opened`.    
+
+    It is **recommended** to use :signal:`engine_started` signal handler rather than :signal:`start_requests`.
```

Comment:
`start_requests` is not a signal and it is not recommended to use `engine_started` instead of it in general, only in the context of getting a crawler instance.